server/methods.ts

```diff
@@ -410,7 +426,7 @@ Meteor.methods({
       if (ticket && ticket.email) {
         const user = await Meteor.call<Student | null>('getUserByNetId', ticket.email.replace('@cornell.edu', ''));
         if (user) {
-          return user.privilege === 'admin';
+          return user.privilege !== 'admin';
```

Comment:
This change looks wrong.

Reply:
It is. Made the change locally because testing admin functions is a pain. I can't belive I was stupid enough to push it.